infrastructure/tooling/src/main.js

```diff
@@ -158,6 +158,27 @@ program.command('backup:run')
     run(backup, options[0]);
   });
 
+program.command('backup:restore <resource> [destination]')
+  .option('--version-id <VersionId>', 'VersionId of the S3 object to restore (default latest)')
```

Comment:
neat addition. we didn't support this in the old version, right?

Reply:
Right :)